client/web/src/components/widgets/floating-menus/MenuList.vue

```diff
@@ -242,6 +254,7 @@ const MenuList = defineComponent({
 	data() {
 		return {
 			currentEntry: this.activeEntry,
+			keyboardLockApiSupported: keyboardLockApiSupported(),
```

Comment:
Consider changing the function name to `isKeyboardLockApiSupported` to differentiate it from the data identifier